packages/mathjax/src/node.js

```
@@ -103,6 +99,10 @@ class Node extends React.Component<Props, *> {
     if (forceUpdate || !this.script) {
       this.setScriptText(text);
     }
+    // As an invariant of above, this shouldn't occur
```

Do you want to actually throw an error since it shouldn't occur? Or log it somehow?

---

I'm basically saying `this.script` got set via `document.createElement` in the above code, though Flow is saying that `this.script` can be nullable (because of type), even though we've set it earlier on.